openhtf/__init__.py

```diff
@@ -114,103 +248,12 @@ def __call__(self, phase_data):
     return self.func(phase_data, **plug_kwargs)
 
 
-class Test(object):
-  """An object that represents an OpenHTF test.
-
-  This object encapsulates the static test state including an ordered tuple of
-  phases to execute.
-
-  Args:
-    *phases: The ordered list of phases to execute for this test.
-    **metadata: Any metadata that should be associated with test records.
-  """
-
-  def __init__(self, *phases, **metadata):
-    """Creates a new Test to be executed.
+def StopOnSigInt(callbacks):
+  """Handles SigInt by calling the given callbacks."""
```

Comment:
`StopOnSigInt`'s docstring looks like it should go with `_Handler`, while `StopOnSigInt` itself should be more like "Create a SIGINT handler..." The point is that there's nothing in `_Handler` that ties it to executor etc. specifically.